app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+  def show
+    @user = User.find(params[:id])
+  end
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to the Sample App!"
+      redirect_to @user
+    else
+      render 'new'
+    end
+  end
+  
+  private
+  
+  def user_params
+    params.require(:user).permit(:name, :email, :password,
+    :password_confirmation)
+    end
```

Comment:
endの位置をdefに揃えてもらうのと、あんまり最後だけ改行はないから1行にまとめるか、引数をすべて1つずつ開業するとかのがいいかな！
```suggestion
  def user_params
    params.require(:user).permit(:name, :email, :password,
    :password_confirmation)
    # params.require(:user).permit(:name, :email, :password, :password_confirmation)
    # params.require(:user)
    #       .permit(:name, 
    #               :email,
    #               :password,
    #               :password_confirmation)
    # 
    # 
  end
```

Reply:
承知しました！
読みやすさにこだわって書いてみます